components/brave_rewards/resources/extension/brave_rewards/_locales/en_GB/messages.json

```diff
@@ -168,11 +168,11 @@
     "description": ""
   },
   "braveContributeTitle": {
-    "message": "Contribute",
```

Comment:
everything except for the en_US messages.json file is auto-generated, so pls remove changes to this file.

Reply:
removed